saleor/graphql/account/mutations/authentication.py

```diff
@@ -356,3 +356,32 @@ def perform_mutation(cls, root, info, **data):
         input_data = data["input"]
         manager = info.context.plugins
         return cls(logout_data=manager.external_logout(input_data, request))
+
+
+class ExternalVerify(BaseMutation):
+    user = graphene.Field(User, description="User assigned to data.")
+    is_valid = graphene.Boolean(
+        required=True,
+        default_value=False,
+        description="Determine if authentication data is valid or not.",
+    )
+    verify_data = graphene.JSONString(description="External data.")
```

Comment:
Maybe `extra_data` is a more proper name for this field?

Reply:
I followed the same pattern for ExternalAuthentication, ExternalRefresh and ExternalVerify. If we need to change it here we should change this for all external mutations